docs/user-guide/resources-properties.md

```diff
@@ -0,0 +1,80 @@
+
+<img src="../img/resource-properties/edit-properties.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+Through the **Edit properties** window the user can perform the following operations:
+
+* Add a **Thumbnail**
+
+* Add **Name** and **Description**
+
+* Add a **Permission** rule
+
+!!! warning
+    The name of a resource is the only mandatory field. Note that is not allowed to choose a name that has already been attributed to another resource.
+
+## Thumbnail
+
+It is possible to add an image as **thumbnail** dropping it or clicking inside the following box:
+
+<img src="../img/resource-properties/thumb.jpg" class="ms-docimage"/>
+
+!!! warning
+    The image to be added must not be larger than 500 kb and its best dimensions are 300x180 px.
+
+## Permission rule
+
+In the **Add a rule...** section you can set one ore more permission rules in order to allow everyone and/or specific groups of users to access the resource. Moreover it is possible to choose between two different ways with witch the selected group can approach the resource:
+
+* **View** the map and save a copy
+
+* **Edit** the map and re-save it
+
+In order to add a rule, the user can select the group and the access modality in the *Add a rule...* section. Once the rule is set, with the  **Add** button <img src="../img/button/add-rule-icon.jpg" class="ms-docbutton"/> it is possible to add it to the **Permissions Groups** list. <br>
+For example, a resource that can be seen by everyone, should have a rule like the following:
+
+<img src="../img/resource-properties/rule_added.jpg" class="ms-docimage" />
+
+Once a rule is set, the user can always remove it through the **Remove** button <img src="../img/button/remove-rule-icon.jpg" class="ms-docbutton"/>. <br>
+How to manage users and groups is a topic present in the [Managing Users](managing-users.md) and [Managing Groups](managing-groups.md) sections.
+
+## Details
+
+Only for map type resource, and only from the **Edit properties** button <img src="../img/button/edit-icon.jpg" class="ms-docbutton"/> in [Homepage](https://mapstore.geo-solutions.it/mapstore/#/), it is possible to add some details to the map. In this case the **Edit properties** window is the following:
+
+<img src="../img/resource-properties/edit-map-properties-panel_details.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> it opens a panel where the user can write the details of the map. 
+
+<img src="../img/resource-properties/details_panel.jpg" class="ms-docimage"/>
+
+The text can be edit and some links and images can be added through the [Text Editor Toolbar](text-editor-details.md). Once the customization is done, the changes can be saved with the **Save** button <img src="../img/button/save_large_button.jpg" class="ms-docbutton"/> and other three buttons appears:
```

Comment:
The text can be **edited**

Once the **editing** is done the **map details** can be saved